src/example/web.js

```diff
@@ -55,13 +55,35 @@ const
 	}],
 
 	// define port (should be an env var in production)
-	port = 5555;
+	PORT = 5555;
```

Comment:
Constant should be at the top, after requires

Reply:
Done